native_client/client.cc

```diff
@@ -173,7 +184,7 @@ main(int argc, char **argv)
       free(result->string);
     }
 
-    if ((argc == 5) && (strncmp(argv[4], "-t", 3) == 0)) {
+    if ((argc == 7) && (strncmp(argv[6], "-t", 3) == 0)) {
```

Comment:
It seems to go weird, and `-t` is not being taken into account: https://tools.taskcluster.net/groups/PjAArtCrS0OIKXZX73sn7g/tasks/HGNuUjlvTn-g8mc_MFFPEw/runs/0/logs/public%2Flogs%2Flive_backing.log#L19589

Reply:
Maybe we should just:
```
if (strncmp(argv[argc - 1], "-t", 3) == 0) {
```

Reply:
That suggestion does work :) https://public-artifacts.taskcluster.net/eQQbpIfvTmWr_H-Ky9dj2A/0/public/benchmark-multi-model.png